2024-11-20 06:20:00,176 ERROR scheduler Skipped queueing frappe.model.utils.link_count.update_link_count because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,183 ERROR scheduler Skipped queueing frappe.desk.form.document_follow.send_daily_updates because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,197 ERROR scheduler Skipped queueing frappe.automation.doctype.auto_repeat.auto_repeat.make_auto_repeat_entry because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,213 ERROR scheduler Skipped queueing frappe.desk.page.backups.backups.delete_downloadable_backups because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,217 ERROR scheduler Skipped queueing frappe.oauth.delete_oauth2_data because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,222 ERROR scheduler Skipped queueing frappe.core.doctype.log_settings.log_settings.run_log_clean_up because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,230 ERROR scheduler Skipped queueing frappe.utils.global_search.sync_global_search because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,235 ERROR scheduler Skipped queueing frappe.model.utils.user_settings.sync_user_settings because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,245 ERROR scheduler Skipped queueing frappe.website.doctype.personal_data_deletion_request.personal_data_deletion_request.process_data_deletion_request because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,260 ERROR scheduler Skipped queueing frappe.email.queue.flush because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,266 ERROR scheduler Skipped queueing frappe.integrations.doctype.google_drive.google_drive.daily_backup because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,276 ERROR scheduler Skipped queueing frappe.social.doctype.energy_point_settings.energy_point_settings.allocate_review_points because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,282 ERROR scheduler Skipped queueing frappe.sessions.clear_expired_sessions because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,288 ERROR scheduler Skipped queueing frappe.website.doctype.personal_data_deletion_request.personal_data_deletion_request.remove_unverified_record because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,296 ERROR scheduler Skipped queueing frappe.desk.doctype.event.event.send_event_digest because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,304 ERROR scheduler Skipped queueing frappe.email.doctype.email_account.email_account.pull because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,315 ERROR scheduler Skipped queueing frappe.integrations.doctype.dropbox_settings.dropbox_settings.take_backups_daily because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,320 ERROR scheduler Skipped queueing frappe.integrations.doctype.google_calendar.google_calendar.sync because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,324 ERROR scheduler Skipped queueing frappe.email.doctype.newsletter.newsletter.send_scheduled_email because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,328 ERROR scheduler Skipped queueing frappe.desk.notifications.clear_notifications because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,336 ERROR scheduler Skipped queueing frappe.desk.form.document_follow.send_hourly_updates because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,345 ERROR scheduler Skipped queueing frappe.email.doctype.notification.notification.trigger_offset_alerts because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,350 ERROR scheduler Skipped queueing frappe.integrations.doctype.google_contacts.google_contacts.sync because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,354 ERROR scheduler Skipped queueing frappe.email.doctype.notification.notification.trigger_daily_alerts because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,365 ERROR scheduler Skipped queueing frappe.website.doctype.web_page.web_page.check_publish_status because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,375 ERROR scheduler Skipped queueing frappe.integrations.doctype.s3_backup_settings.s3_backup_settings.take_backups_daily because it was found in queue for fabrixcel.localhost
2024-11-20 06:20:00,381 ERROR scheduler Skipped queueing frappe.email.doctype.auto_email_report.auto_email_report.send_daily because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,157 ERROR scheduler Skipped queueing frappe.integrations.doctype.google_calendar.google_calendar.sync because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,171 ERROR scheduler Skipped queueing frappe.oauth.delete_oauth2_data because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,205 ERROR scheduler Skipped queueing frappe.twofactor.delete_all_barcodes_for_users because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,218 ERROR scheduler Skipped queueing frappe.email.doctype.newsletter.newsletter.send_scheduled_email because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,236 ERROR scheduler Skipped queueing frappe.model.utils.user_settings.sync_user_settings because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,257 ERROR scheduler Skipped queueing frappe.email.doctype.email_account.email_account.notify_unreplied because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,310 ERROR scheduler Skipped queueing frappe.email.doctype.notification.notification.trigger_offset_alerts because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,327 ERROR scheduler Skipped queueing frappe.email.queue.flush because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,333 ERROR scheduler Skipped queueing frappe.website.doctype.web_page.web_page.check_publish_status because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,357 ERROR scheduler Skipped queueing frappe.email.doctype.email_account.email_account.pull because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,379 ERROR scheduler Skipped queueing frappe.model.utils.link_count.update_link_count because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,413 ERROR scheduler Skipped queueing frappe.website.doctype.personal_data_deletion_request.personal_data_deletion_request.process_data_deletion_request because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,421 ERROR scheduler Skipped queueing frappe.monitor.flush because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,431 ERROR scheduler Skipped queueing frappe.desk.page.backups.backups.delete_downloadable_backups because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,438 ERROR scheduler Skipped queueing frappe.utils.global_search.sync_global_search because it was found in queue for fabrixcel.localhost
2024-11-20 06:32:00,455 ERROR scheduler Skipped queueing frappe.desk.form.document_follow.send_hourly_updates because it was found in queue for fabrixcel.localhost
2024-11-20 09:16:00,942 ERROR scheduler Skipped queueing frappe.utils.global_search.sync_global_search because it was found in queue for fabrixcel.localhost
2024-11-20 09:16:00,972 ERROR scheduler Skipped queueing frappe.website.doctype.web_page.web_page.check_publish_status because it was found in queue for fabrixcel.localhost
2024-11-20 09:16:00,990 ERROR scheduler Skipped queueing frappe.automation.doctype.reminder.reminder.send_reminders because it was found in queue for fabrixcel.localhost
2024-11-20 09:16:01,169 ERROR scheduler Skipped queueing frappe.monitor.flush because it was found in queue for fabrixcel.localhost
2024-11-20 09:28:00,551 ERROR scheduler Exception in Enqueue Events for Site fabrixcel.localhost
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 106, in enqueue_events_for_site
    if is_scheduler_inactive():
       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 150, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 163, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 864, in get_single_value
    ).run()
      ^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 84, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 214, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 119, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
                                                             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
                  ^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_99b112c57448a052'@'172.21.0.3' (using password: YES)")
2024-11-20 09:32:00,604 ERROR scheduler Exception in Enqueue Events for Site fabrixcel.localhost
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 106, in enqueue_events_for_site
    if is_scheduler_inactive():
       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 150, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 163, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 864, in get_single_value
    ).run()
      ^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 84, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 214, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 119, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
                                                             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
                  ^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_99b112c57448a052'@'172.21.0.3' (using password: YES)")
2024-11-20 09:36:00,611 ERROR scheduler Exception in Enqueue Events for Site fabrixcel.localhost
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 106, in enqueue_events_for_site
    if is_scheduler_inactive():
       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 150, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 163, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 864, in get_single_value
    ).run()
      ^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/query_builder/utils.py", line 84, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 214, in sql
    self.connect()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/database.py", line 119, in connect
    self._conn: "MariadbConnection" | "PostgresConnection" = self.get_connection()
                                                             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 107, in get_connection
    conn = self._get_connection()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 113, in _get_connection
    return self.create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/database.py", line 116, in create_connection
    return pymysql.connect(**self.get_connection_settings())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 361, in __init__
    self.connect()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 669, in connect
    self._request_authentication()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 957, in _request_authentication
    auth_packet = self._read_packet()
                  ^^^^^^^^^^^^^^^^^^^
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/connections.py", line 775, in _read_packet
    packet.raise_for_error()
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/protocol.py", line 219, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe/frappe-bench/env/lib/python3.11/site-packages/pymysql/err.py", line 150, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_99b112c57448a052'@'172.21.0.3' (using password: YES)")
